### DISCUSSION TITLE
No prefix in specific channel

### DISCUSSION THREAD

**Discussion by romain-vdw** [jagrosh/MusicBot#1716] [Ideas]
Is there any way to let there be a channel where you don't need a prefix but only for the "play" command?
Like you could put a link/title in the channel and the bot would play the song without needing to put "play" in front of it.

**Comment by deepakpaswan12869-art** [comment]
Noprefix

**Comment by deepakpaswan12869-art** [comment]
Discord free giveaway + noprefix bot

**Comment by chillybeam90-eng** [comment]
[nba locker codes](https://2k26lockercodes.com/) offers one of the deepest player progression systems in sports gaming, giving fans the opportunity to create a unique basketball star and guide them through a complete professional journey. From the moment you create your MyPLAYER, every decision matters. Your build, attributes, badges, animations, and gameplay style all influence how successful you become on the virtual court. While many players focus on reaching a high overall rating as quickly as possible, true success comes from understanding the progression system and making smart choices throughout your career.
